pkg/cel/ast/inspector.go

```diff
@@ -256,7 +256,7 @@ func (a *Inspector) inspectCall(call *exprpb.Expr_Call, currentPath string) Expr
 		inspection.FunctionCalls = append(inspection.FunctionCalls, FunctionCall{
 			Name: fmt.Sprintf("%s.%s", a.exprToString(call.Target), call.Function),
 		})
-	} else if !isInternalFunction(call.Function) {
+	} else if !a.env.HasFunction(call.Function) {
```

Comment:
Would this also cover the function added via #506 ?

Reply:
Yes! The test below has `random.seededString` in the list of tested functions.